README.md

```diff
@@ -11,7 +11,13 @@ A website for a web development course
 
 ## Live Demo
 
--
+Click the [Link](https://capstone-wecode.netlify.app/) to see a Live Demo to this project.
+
+## Live Video Recording
```

Comment:
- [ ] [OPTIONAL] Great improvement based on the previous reviewers' suggestions. In addition to what you have done so far, **I would suggest that you add a screenshot of your website landing page**. You can add it above the `Live Demo` section. This gives visitors a quick preview of what your website looks like 😉 👍

Reply:
Done!!! @BertrandConxy